app.py

```diff
@@ -7,6 +7,7 @@
 
 # These are the extension that we are accepting to be uploaded
 ALLOWED_EXTENSIONS = set(['png', 'jpg', 'jpeg'])
+ALLOWED_EXTENSIONS_LOGO = set([' svg'])
```

Comment:
This part looks wrong also in the existing code. We have a set with items like "png" above and here a set with a space and below a set with points. I would say make it uniform like line 9

Reply:
Ok I will do that

Reply:
The tested the build, and then this files was made like wallpapers was made in the earlier version
I will delete it in the next commit